docs/index.md

```diff
@@ -6,69 +6,87 @@ This page is the top level entry point into the Substrate documentation.
+  Substrate makes it possible to supply custom consensus engines and also ships with several consensus mechanisms that
+  have been built on top of [Web3 Foundation research](https://w3f-research.readthedocs.io/en/latest/index.html).
+- **Remote procedure calls** allow blockchain users to interact with the network. Substrate provides HTTP and WebSocket
+  RPC servers.
+- **Telemetry** metrics are exposed by way of an embedded [Prometheus](https://prometheus.io/) server.
 
 ## Usage
 
 ![Technical Freedom vs Development Ease](assets/technical-freedom.png)
 
 Substrate is designed to be used in one of three ways:
 
-1. **With the Substrate Node**: You can run the pre-designed Substrate Node and configure it with a
-   genesis block that includes the default node runtime. In this case, you just need to configure a
-   JSON file and launch your own blockchain. This affords you the least amount of customization,
-   only allowing you to change the genesis parameters of the included runtime modules such as:
-   balances, staking, block-period, fees, governance, etc... For a tutorial on doing this, see
-   [Start a Private Network with Substrate](tutorials/start-a-private-network/index.md).
-
-2. **With the Substrate FRAME**: You can easily create your own custom blockchain using the FRAME. This affords
-   you a very large amount of freedom over your own blockchain's logic, letting you change
-   datatypes, select from the library of modules, and add your own custom modules. Much can be
-   changed without touching the block-authoring logic since it is directed through on-chain logic.
-   If this is the case, then the existing Substrate binary can be used for block authoring and
-   syncing. If the block authoring logic needs to be modified, then a new block-authoring binary
-   must be built as a separate project and used by validators. This is how the Polkadot relay chain
-   is built and should suffice for almost all needs in the near future. For a tutorial on this, see
+1. **With the Substrate Node**: You can run the pre-designed Substrate Node and configure it with a genesis block that
+   includes the default node runtime. In this case, you just need to configure a JSON file and launch your own
+   blockchain. This affords you the least amount of customization, only allowing you to change the genesis parameters of
+   the included runtime modules such as: balances, staking, block-period, fees, governance, etc... For a tutorial on
+   doing this, see [Start a Private Network with Substrate](tutorials/start-a-private-network/index.md).
+
+2. **With Substrate FRAME**: You can easily create your own custom runtime using FRAME. This affords you a very large
+   amount of freedom over your own blockchain's logic, letting you change data types, select from the library of
+   modules, and add your own custom modules. Much can be changed without touching the block-authoring logic since it is
```

Comment:
```suggestion
2. **With Substrate FRAME**: You can easily create your own custom runtime using Framework for Runtime Aggregation of Modularized Entities (FRAME). This affords you a very large
   amount of freedom over your own blockchain's logic, letting you change data types, select from the library of
   pallets, or even add your own pallets. Much can be changed without touching the block-authoring logic since it is
```

Reply:
just to make the task of replacement easier 😄